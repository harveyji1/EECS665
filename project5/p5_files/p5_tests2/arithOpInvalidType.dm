main : () void {
    a : bool = true;
    b : bool;
	b = a + true;
}
