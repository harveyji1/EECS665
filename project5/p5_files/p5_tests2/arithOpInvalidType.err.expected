FATAL [4,6]-[4,7]: Arithmetic operator applied to invalid operand
FATAL [4,10]-[4,14]: Arithmetic operator applied to invalid operand
Type Analysis Failed
